apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -998,4 +1002,177 @@ describe('LIVECHAT - rooms', function () {
 			expect(body.messages[1]).to.have.property('username', visitor.username);
 		});
 	});
+
+	describe('livechat/room.saveInfo', () => {
```

Comment:
Can you add more tests for the cases of guestData/roomData not being sent, for sending them as other but objects, as sending invalid values on the keys, etc.

Reply:
For sure :smile: Added a couple of more tests to validate the payloads